src/main/java/com/aws/greengrass/clientdevices/auth/certificate/usecases/RegisterCertificateAuthorityUseCase.java

```diff
@@ -21,7 +21,7 @@
 import javax.inject.Inject;
 
 public class RegisterCertificateAuthorityUseCase
-        implements UseCases.UseCase<Void, Void> {
+        implements UseCases.UseCase<Exception, Void> {
```

Comment:
Interesting take. Maybe we can wrap these so exceptions never get thrown from use cases (but without requiring every use case to catch Exception). Something for later